<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolDialog</class>
 <widget class="QDialog" name="ToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>321</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="checkstackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>321</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="ping">
    <widget class="QPushButton" name="pingButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/images/images/button_normal.png);
	color: rgb(175, 210, 255);
}
QPushButton:focus{
	border-image: url(:/images/images/button_pressed.png);
	color: rgb(175, 210, 255);
}
QPushButton:pressed{
	border-image: url(:/images/images/button_click.png);
	color: rgb(175, 210, 255);
}

</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetIP">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>199</x>
       <y>10</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/images/combox_bg.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
    <widget class="QLabel" name="pinglabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>12</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: rgb(5, 106, 183);</string>
     </property>
     <property name="text">
      <string> Host Name/IP:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="pingresulttextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="statuslabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: rgb(5, 106, 183);</string>
     </property>
     <property name="text">
      <string>Status :</string>
     </property>
    </widget>
    <widget class="QLabel" name="wakeuplabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: rgb(5, 106, 183);</string>
     </property>
     <property name="text">
      <string>Host PC MAC Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="wakeupMAC">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/images/combox_bg.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string>HH:HH:HH:HH:HH:HH; </string>
     </property>
    </widget>
    <widget class="QPushButton" name="wakeupButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/images/images/button_normal.png);
	color: rgb(175, 210, 255);
}
QPushButton:focus{
	border-image: url(:/images/images/button_pressed.png);
	color: rgb(175, 210, 255);
}
QPushButton:pressed{
	border-image: url(:/images/images/button_click.png);
	color: rgb(175, 210, 255);
}

</string>
     </property>
     <property name="text">
      <string>Wakeup</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="log">
    <widget class="QPushButton" name="logsaveButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>270</y>
       <width>128</width>
       <height>26</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/images/images/button_normal.png);
	color: rgb(175, 210, 255);
}
QPushButton:focus{
	border-image: url(:/images/images/button_pressed.png);
	color: rgb(175, 210, 255);
}
QPushButton:pressed{
	border-image: url(:/images/images/button_click.png);
	color: rgb(175, 210, 255);
}

</string>
     </property>
     <property name="text">
      <string>Save to USB</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="logtextBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>231</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>128</width>
       <height>26</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/images/images/button_normal.png);
	color: rgb(175, 210, 255);
}
QPushButton:focus{
	border-image: url(:/images/images/button_pressed.png);
	color: rgb(175, 210, 255);
}
QPushButton:pressed{
	border-image: url(:/images/images/button_click.png);
	color: rgb(175, 210, 255);
}

</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkstackedWidget</tabstop>
  <tabstop>targetIP</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>pingresulttextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
